scripts/import/import_all_sv_data.pl

```diff
@@ -128,18 +128,17 @@ =head1 CONTACT
 
 my %display_name = ( 'COSMIC' => 'https://cancer.sanger.ac.uk/cosmic/' );
 
-my $add             = '';
```

Comment:
The reason for the `$add` was to have an easy way to do a test import on test tables without renaming manually tables in the db. It would be nice to keep this feature in. Ideally this option would be accessible from outside the script as a real option.